src/engine.rs

```diff
@@ -224,6 +228,7 @@ impl Reedline {
             hide_hints: false,
             validator,
             use_ansi_coloring: true,
+            cwd: "".to_string(),
```

Comment:
Just to make it clear - I don't think an empty string should be the default. I think making it an `Option` and then grabbing process working directory if it's `None` is probably the best way to go